justfile

```diff
@@ -15,8 +15,11 @@ public_coordinator_http_port := "80"
 # command to get the local IP of this machine
 get_local_ip := if os() == "linux" {
  "ip -o route get to 1 | sed -n 's/.*src \\([0-9.]\\+\\).*/\\1/p'"
+} else if os() == "macos" {
+ "ipconfig getifaddr en0"
```

Comment:
I'm not sure whether this will work on all the machines - does the current solution not work for you? 
are you using wifi or ethernet?

Reply:
there's little detail in your commit message - it would have been easier if you described what didn't work for you.

Reply:
Does the old solution work for you?

`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\\.){3}[0-9]*'` does not return anything for me.

I'm happy to drop this commit.